GOVERNANCE.md

```diff
@@ -52,40 +48,14 @@ and so on.
 
 The current committers are (in addition to the TC members above):
 
-* [@hugs](https://github.com/hugs)
-* [@filmaj](https://github.com/filmaj)
-* [@aluedeke](https://github.com/aluedeke)
-* [@sebv](https://github.com/sebv)
-* [@vgrigoruk](https://github.com/vgrigoruk)
-* [@DylanLacey](https://github.com/DylanLacey)
-* [@maudineormsby](https://github.com/maudineormsby)
-* [@titusfortner](https://github.com/titusfortner)
-* [@bayandin](https://github.com/bayandin)
-* [@jdeff](https://github.com/jdeff)
-* [@moizjv](https://github.com/moizjv)
-* [@gempesaw](https://github.com/gempesaw)
-* [@Jonahss](https://github.com/Jonahss)
-* [@JamieEdge](https://github.com/JamieEdge)
 * [@saikrishna321](https://github.com/saikrishna321)
-* [@sravanmedarapu](https://github.com/sravanmedarapu)
-* [@peepa](https://github.com/peepa)
-* [@scottdixon](https://github.com/scottdixon)
-* [@drpy](https://github.com/drpy)
-* [@Astro03](https://github.com/Astro03)
-* [@sbonebrake](https://github.com/sbonebrake)
 * [@SrinivasanTarget](https://github.com/SrinivasanTarget)
-* [@triager](https://github.com/triager)
-* [@ddkjin](https://github.com/ddkjin)
-* [@stuartbrussell-intuit](https://github.com/stuartbrussell-intuit)
-* [@vmaxim](https://github.com/vmaxim)
 * [@Dan-Maor](https://github.com/Dan-Maor)
 * [@laolubenson](https://github.com/laolubenson)
 * [@Dor-bl](https://github.com/Dor-bl)
-* [@boneskull](https://github.com/boneskull)
 * [@mwakizaka](https://github.com/mwakizaka)
 * [@rerorero](https://github.com/rerorero)
 * [@tomriddly](https://github.com/tomriddly)
-* [@eglitise](https://github.com/eglitise)
```

Comment:
Maybe it will be nice to group the committers per project (in case they don't work on multiple ones)

Reply:
Good idea, maybe I can just allow people to write what they maintain beside their names in brackets. Groups are difficult since some people are in multiple groups.